src/index.ts

```diff
@@ -12,30 +17,76 @@ import { FileReaderGame } from './types';
  * @returns
  */
 export async function main(path: string) {
+  console.time("Total Execution Time");
+
+  console.time("Task 1: FileReader")
   const gamesGenerator = gameChunks(path);
   const games: FileReaderGame[] = [];
+  let gameCounter = 0;
   for await (const game of gamesGenerator) {
+    gameCounter++;
+    if (gameCounter % 20 == 0) {
+      console.log('number of games analyzed: ', gameCounter);
+    }
     games.push(game);
+
+    // const siteLink = game.metadata[1].match(/"(.*?)"/)[1];
+    // console.log(`lichess link to game played: ${siteLink}`);
```

Comment:
rm or uncomment?

Reply:
can remove

Reply:
actually can we just keep this in
sometimes i want all the sitelinks in one place listed if i wanna take a look at them

Reply:
one exception to your "no commented logs rule" pls ;)

Reply:
make it an optional flag if it's really impt